packages/react/src/patterns/blocks/LeadSpaceBlock/__stories__/LeadSpaceBlock.stories.js

```diff
@@ -0,0 +1,120 @@
+import {
+  object,
+  select,
+  withKnobs,
+  text
+} from '@storybook/addon-knobs';
+import LeadSpaceBlock from '../LeadSpaceBlock.js';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+import {HorizontalRule} from "../../../../components/HorizontalRule";
+
+storiesOf('Patterns (Blocks)|LeadSpaceBlock', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+
+    const title = 'Continuous delivery';
+
+    const heading= text(
+      'Heading (required)',
+      'Innovate like a startup and scale for the enterprise '
+    );
+
+    const copy = `Automate your software release process with continuous delivery (CD)—the most 
+    critical part of adopting DevOps. Build, test, and deploy code changes quickly, 
+    ensuring software is always ready for deployment.`;
+
+    const mediaType = select(
+      'mediaType (optional)',
+      ['image', 'video', 'none'],
+      'image'
+    );
+
+    const video = {
+      videoId: '0_uka1msg4',
+      showDescription: true,
```

Comment:
this has been updated to `showCaption` instead

Reply:
@annawen1 added caption there.